src/Fabulous/Array.fs

```diff
@@ -258,8 +258,8 @@ module StackAllocatedCollections =
 
     [<Struct; NoComparison; NoEquality>]
     type StackArray3<'v> =
-        | Few of data: struct (Size * 'v * 'v * 'v)
         | Many of arr: 'v array
+        | Few of data: Size * 'v * 'v * 'v
```

Comment:
Removes indirection and decreases total DU struct size with some type parameters.